compute_tools/src/schema.rs

```diff
@@ -0,0 +1,116 @@
+use compute_api::{
+    responses::SchemaObjects,
+    spec::{Database, Role},
+};
+use futures::Stream;
+use postgres::{Client, NoTls};
+use std::{path::Path, process::Stdio, result::Result, sync::Arc};
+use tokio::{
+    io::{AsyncBufReadExt, BufReader},
+    process::Command,
+    task,
+};
+use tokio_stream::{self as stream, StreamExt};
+use tokio_util::codec::{BytesCodec, FramedRead};
+use tracing::warn;
+
+use crate::{
+    compute::ComputeNode,
+    pg_helpers::{get_existing_dbs, get_existing_roles},
+};
+
+pub async fn get_schema_objects(compute: &Arc<ComputeNode>) -> anyhow::Result<SchemaObjects> {
```

Comment:
It would be good to name the functions similarly

Reply:
done, schema_dump -> get_schema_ddl